src/helpers/checkAreaHelpers.js

```diff
@@ -15,7 +15,18 @@ export function getAlignedGLText(currentProjectToolsSelectedGL, contextId, bible
       }
     }
   }
-  return contextId.quote;
+  const origLangQuote = getQuoteAsString(contextId.quote);
+  const message = translate("quote_invalid", {quote: origLangQuote});
+  return message;
+}
+
+export function getQuoteAsString(quote) {
+  if (Array.isArray(quote)) {
+    return quote.map(({word}) => word).join(' ');
```

Comment:
I don't think you added support for punctuation

Reply:
Hmm...I copied this from tC - thought this was what you and Rich had agreed on.  OK found the method flattenQuote().

Reply:
OK, moved method from tCore to this module for sharing.

Reply:
Does this make it more standard function? Cool.